src/miner.cpp

```diff
@@ -92,7 +92,7 @@ CBlockTemplate* CreateNewBlock(const CChainParams& chainparams, const CScript& s
 CBlockTemplate* CreateNewBlock(const CChainParams& chainparams, const CScript& scriptPubKeyIn,bool blockstreamCoreCompatible)
 {
     // Create new block
-    unique_ptr<CBlockTemplate> pblocktemplate(new CBlockTemplate());
```

Comment:
Are you asking did I review this manually?  Yes, and I would like you to do so if you are not.  I can easily merge the entire 0.13 core changeset into BU as I was doing for 0.12 branch.  For this piecemeal approach to have value, we need to look at what we are doing.

I will merge this even with std:: because its not a big deal to have or not have them.